docs/guide/advanced/deployment/self-hosted.md

```diff
@@ -6,7 +6,7 @@ This guide will explain how you can self-host a Nuxt3 application running NuxtAu
 
 When deploying the Auth.JS provider, the application must be informed what URL it is running at. This is to properly determine callback urls when navigating users to external OAuth providers. Depending on your setup, NuxtAuth allows you to set this value at either [**Runtime**](https://nuxt.com/docs/guide/going-further/hooks#app-hooks-runtime) or [**Buildtime**](https://nuxt.com/docs/guide/going-further/hooks#nuxt-hooks-build-time).
 
-- **Runtime:** Set the `AUTH_ORIGIN` environment variable.
+- **Runtime:** Set the `NUXT_AUTH_ORIGIN` environment variable.
```

Comment:
Is the default now set to: `NUXT_AUTH_ORIGIN` or just `AUTH_ORIGIN`? If it gets changed to `NUXT_AUTH_ORIGIN`, I belive it is still a breaking change. Otherwise are the docs just outdated here?

Reply:
`AUTH_ORIGIN` support remains, but `NUXT_AUTH_ORIGIN` is preferred. Therefore, it is suggested to use `NUXT_AUTH_ORIGIN` for new users

Reply:
The default looked to me like `AUTH_ORIGIN` like we have now, isn't it?

Reply:
> AUTH_ORIGIN support remains, but NUXT_AUTH_ORIGIN is preferred. Therefore, it is suggested to use NUXT_AUTH_ORIGIN for new users

Okay! So by default both are supported?

Reply:
Yes :)

There is this line in the config:
```
originEnvKey: 'AUTH_ORIGIN',
```
This means that the ENV will be `NUXT_AUTH_ORIGIN` or `AUTH_ORIGIN` by default. The user will be able to change it if they want to